docs/framework/additional-apis/system.net.mime.mailbnfhelper.md

```diff
@@ -0,0 +1,121 @@
+---
+title: MailBnfHelper class (System.Net)
+ms.date: 06/12/2020
+ms.technology: dotnet-networking
+topic_type: 
+  - apiref
+api_name: 
+  - System.Net.Mime.MailBnfHelper
+  - System.Net.Mime.MailBnfHelper.Ascii7bitMaxValue
+  - System.Net.Mime.MailBnfHelper.Atext
+  - System.Net.Mime.MailBnfHelper.CR
+  - System.Net.Mime.MailBnfHelper.Ctext
+  - System.Net.Mime.MailBnfHelper.Dot
+  - System.Net.Mime.MailBnfHelper.EndComment
+  - System.Net.Mime.MailBnfHelper.LF
+  - System.Net.Mime.MailBnfHelper.Space
+  - System.Net.Mime.MailBnfHelper.StartComment
+  - System.Net.Mime.MailBnfHelper.Tab
+api_location: 
+  - System.dll
+api_type: 
+  - Assembly
+---
+# MailBnfHelper class
+
+Contains utility methods for parsing internet message-formatted strings. This class cannot be inherited.
```

Comment:
Is this accurate? It's not listed as sealed.

Reply:
The [wiki guide](https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Class) says to add that to all static classes.